### ISSUE TITLE
No read the chat and channel point 

### ISSUE THREAD

**Issue by 132n3rd** [twitchdev/issues#343]
**Brief description**
i'm follow this italian channel  on twitch ATEXCHANNEL, with my channel  123n3rd and i having issue with the chat.

I see only few peaple talk in chat, not all peaple that are interact with the chat. 
 I have another issue, i can't see channel points for interact too.

the channel, iìm having issue is this one https://www.twitch.tv/atexchannel.

i put a creeenshot of the chat lot of peaple chatting but i can see only few peaple, and as i say before non channel points 

![Issue on ATEXCHANNEL chat](https://user-images.githubusercontent.com/80067107/110020898-64e8aa00-7d2a-11eb-8053-119396449f94.jpg)

tell me if i can act in any way or have to wait untill dhe bug get fixed.

Thanks

123n3rd

**Comment by BarryCarlyon** [comment]
atexchannel doesn't have channel points enabled on their channel

as per

https://help.twitch.tv/s/article/channel-points-guide?language=en_US

> Channel Points is available to all Twitch Affiliates & Partners.

atexchannel is neither of these.

**Comment by 132n3rd** [comment]
that's pretty wired. Other peaple still using channel point from him, he have about 2000 followers and few subscription too using Amazon.
But the most important issue is i'm not able to see all the peaple in chat just few about 30 peaple in it, and i only can see only 2 peaple chatting with Him

**Comment by BarryCarlyon** [comment]
>  few subscription too using Amazon.

Not sure what you mean.

> Other peaple still using channel point from him,

Sounds like this channel has it's own points system via the channel bot

![image](https://user-images.githubusercontent.com/20999/110331355-ea17db80-8016-11eb-883e-9c034b2aefc1.png)

The users API states they are not affiliate or partner and thus don't have subscriptions from Twitch or channel points enabled on the channel

> But the most important issue is i'm not able to see all the peaple in chat just few about 30 peaple in it, and i only can see only 2 peaple chatting with Him

Your screenshot shows 3 viewers. So I'd expect up to 3 chatters and any bots.

The number of people in the "chat members list" doesn't correspond to people actually present/watching the stream/actively in chat.

**Comment by 132n3rd** [comment]
I try to explain myself better.
During a live show, only I see a smaller number of participants and chat interactions, while other users and Atexchannel itself have the complete view of everything. As soon as possible, I will try to make a short video illustrating the problem.

**Comment by BarryCarlyon** [comment]
> while other users and Atexchannel itself have the complete view of everything

I went to their channel and didn't see channel points or subscriptions enabled.
I also checked the API and determined they are not elibile for these things.

> During a live show, only I see a smaller number of participants and chat interactions, while other users and Atexchannel itself have the complete view of everything.

This sounds like they are streaming to multiple sites and thus the feature sets differ.

So if they are streaming to youtube and twitch, they are talking to people in YouTube's chat

Either way it's not a bug and thus is irrelevant to this bug tracker
